src/main/java/twilight/Storage.java

```diff
@@ -0,0 +1,77 @@
+package twilight;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * Handles creation of, reading from, and writing to textfile which stores the tasklist.
+ */
+public class Storage {
+    private final String filePath;
+
+
+    /**
+     * Instantiates Storage.
+     *
+     * @param filePath Path to the storage file from root.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
```

Comment:
I might be wrong, but shouldn't there be an empty line after the description and before the @param?